frontend/app/src/utils/text.ts

```diff
@@ -0,0 +1,11 @@
+// john doe => John doe
```

Comment:
nitpick: can you make the comments to JSDoc so that we can have a proper documentation popup in webstorm? :smiley:

Reply:
Updated @kelsos